### PR TITLE
Adds title on the user images to show corresponding names, fixes #494

### LINKED ISSUE #494: Workitem list view onhover assignee icon, assignee name not showup
Workitem list view on hover assignee icon, assignee name not showup
screenshot from 2016-12-12 11-13-36

### PR DESCRIPTION
**Pull request by debloper**
Signed-off-by: Soumya Deb <debloper@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/almighty/almighty-ui/599)

### CODE REVIEW

File: src/app/work-item/work-item-list/work-item-detail/work-item-comment/work-item-comment.component.html [hunk 1]
```diff
@@ -12,6 +12,7 @@
                 <div>
                     <img id="{{'comment_avatar_' + counter}}"
                       class="user-assign-avatar pull-left" 
+                      title="{{comment.relationalData.creator?.attributes?.fullName}}"
```
Comment (vikram-raj) [inline src/app/work-item/work-item-list/work-item-detail/work-item-comment/work-item-comment.component.html]: We are not using the default tooltip provided by HTML. 
We are using the ng2-bootstrap tooltip.
Please refer to this PR https://github.com/almighty/almighty-ui/pull/569.

Comment (debloper) [inline src/app/work-item/work-item-list/work-item-detail/work-item-comment/work-item-comment.component.html]: Thanks for the heads-up, updating the PR

**Comment by joshuawilson** [comment]
@debloper please rebase

**Comment by debloper** [comment]
[test]

**Comment by nimishamukherjee** [comment]
[test]

**Comment by nimishamukherjee** [comment]
[test]

**Comment by joshuawilson** [comment]
@debloper there are tests failing on the ci-server. Can you please look into this.

**Comment by nimishamukherjee** [comment]
[test]

**Comment by nimishamukherjee** [comment]
[test]

**Comment by nimishamukherjee** [comment]
[test]

**Comment by nimishamukherjee** [comment]
[test]

**Comment by nimishamukherjee** [comment]
[test]

**Comment by nimishamukherjee** [comment]
[test]

**Comment by debloper** [comment]
Hang on... checking again! >_>

**Pull request merged**